packages/mdc-line-ripple/README.md

```diff
@@ -50,14 +42,20 @@ Mixin | Description
 --- | ---
 `mdc-line-ripple-color($color)` | Customizes the color of the line ripple when active.
 
-### `MDCLineRipple`
+## `MDCLineRipple` Properties and Methods
+
+### Methods
 
 Method Signature | Description
 --- | ---
 `activate() => void` | Proxies to the foundation's `activate()` method.
 `deactivate() => void` | Proxies to the foundation's `deactivate()` method.
 `setRippleCenter(xCoordinate: number) => void` | Proxies to the foundation's `setRippleCenter(xCoordinate: number)` method.
 
+## Usage Within Frameworks
+
+If you are using a JavaScript framework, such as React or Angular, you can create a `MDCTextField` for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](../../docs/integrating-into-frameworks.md).
```

Comment:
`MDCTextField` should be Line ripple (copypaste, and <COMPONENT_NAME> doesn't refer to the literal JS name, that's <JS_COMPONENT_NAME>)